pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# References <!-- Link to the Jira task -->
+[PROJ-XXX](https://frontkom.atlassian.net/browse/PROJ-XXX)
```

Comment:
I think we should use the auto-links, so we can keep this simple:

```suggestion
PROJ-XXX
```

Reply:
will it really auto link though?

Reply:
If configured yes and in on the commit text as well. :) I've set it up on several projects:
![image](https://github.com/frontkom/.github/assets/401819/dbe72b57-8ea7-4be6-8b4e-9c55955c139f)